apps/admin-portal/src/api/application.ts

```diff
@@ -211,3 +211,144 @@ export const getApplicationList = (limit: number, offset: number): Promise<any>
             return Promise.reject(error);
         });
 };
+
+/**
+ * Gets the available inbound protocols.
+ *
+ * @param customOnly If true only returns custom protocols.
+ */
+export const getAvailableInboundProtocols = (customOnly: boolean): Promise<any> => {
+    const requestConfig = {
+        headers: {
+            "Accept": "application/json",
+            "Access-Control-Allow-Origin": GlobalConfig.clientHost,
+            "Content-Type": "application/json"
+        },
+        method: HttpMethods.GET,
+        url: ServiceResourcesEndpoint.applications + "/meta/inbound-protocols?customOnly=" + customOnly
+    };
+
+    return httpClient.request(requestConfig)
+        .then((response) => {
+            if (response.status !== 200) {
```

Comment:
Shall we remove this if condition as the then block will only be executed when the request is successful?

Reply:
Let's keep them for now. Currently, we are only expecting 200 status